kronofoto/static/assets/js/main-dev.js

```diff
@@ -1,18 +1,114 @@
 import HTMX from "./htmx.js"
 import timeline from "./timeline.js"
-import {installButtons, toggleMenu, markerDnD, toggleLogin} from "./lib.js"
+import {
+  installButtons,
+  toggleMenu,
+  markerDnD,
+  toggleLogin,
+  initGalleryNav,
+  autoplayStart,
+  autoplayStop,
+  timelineForward,
+  timelineBackward,
+  timelineZipForward,
+  timelineZipBackward,
+  timelineCrawlForward,
+  timelineCrawlForwardRelease,
+  timelineCrawlBackward,
+  timelineCrawlBackwardRelease,
+  moveTimelineCoin,
+  dropTimelineCoin }
+  from "./lib.js"
+
 window.toggleLogin = toggleLogin
 
 const htmx = HTMX(document)
+window.htmx = htmx
```

Comment:
I think this method of saving the htmx instance will not work if more than one web component is used on a page.

Reply:
I forget why I did that. I think I needed to use it in lib.js. Maybe there's a better way to pass it to that file.